demo/src/api.js

```diff
@@ -31,6 +31,10 @@ module.exports.handler = elmServerless.httpApi({
   // Your elm app is the handler
   handler: elm.API,
 
+  interop: {
+    getRandom: upper => Promise.resolve(Math.floor(Math.random() * upper)),
+  },
```

Comment:
On the JavaScript side, the user writes an interop function handler. The interop call was `GetRandom 10`, so the function is named `getRandom`